src/core/src/main/java/org/apache/jmeter/gui/util/JSyntaxTextArea.java

```diff
@@ -44,20 +49,27 @@
 public class JSyntaxTextArea extends RSyntaxTextArea {
 
     private static final long serialVersionUID = 211L;
-    private static final Logger log              = LoggerFactory.getLogger(JSyntaxTextArea.class);
+    private static final Logger log = LoggerFactory.getLogger(JSyntaxTextArea.class);
 
+    private static final Theme DARKLAF_THEME = new DarklafRSyntaxTheme();
```

Comment:
This line seems to cause `HeadlessException`, see https://travis-ci.org/github/apache/jmeter/jobs/725243535#L844

Is static theme really needed here?


```
    java.lang.ExceptionInInitializerError
        at org.apache.jmeter.assertions.jmespath.gui.JMESPathAssertionGui.buildPanel(JMESPathAssertionGui.java:78)
        at org.apache.jmeter.assertions.gui.JSONPathAssertionGui.init(JSONPathAssertionGui.java:71)
        at org.apache.jmeter.assertions.gui.JSONPathAssertionGui.<init>(JSONPathAssertionGui.java:63)
        at org.apache.jmeter.assertions.jmespath.gui.JMESPathAssertionGui.<init>(JMESPathAssertionGui.java:59)
        at org.apache.jmeter.assertions.jmespath.gui.JMESPathAssertionGuiTest.testInit(JMESPathAssertionGuiTest.java:28)
        Caused by: java.awt.HeadlessException
            at java.desktop/javax.swing.text.JTextComponent.checkDragEnabled(JTextComponent.java:698)
            at java.desktop/javax.swing.text.JTextComponent.setDragEnabled(JTextComponent.java:692)
            at org.fife.ui.rtextarea.RTextArea.init(RTextArea.java:918)
            at org.fife.ui.rsyntaxtextarea.RSyntaxTextArea.init(RSyntaxTextArea.java:1988)
            at org.fife.ui.rtextarea.RTextAreaBase.<init>(RTextAreaBase.java:82)
            at org.fife.ui.rtextarea.RTextArea.<init>(RTextArea.java:188)
            at org.fife.ui.rsyntaxtextarea.RSyntaxTextArea.<init>(RSyntaxTextArea.java:351)
            at com.github.weisj.darklaf.extensions.rsyntaxarea.DarklafRSyntaxTheme.<init>(DarklafRSyntaxTheme.java:40)
            at org.apache.jmeter.gui.util.JSyntaxTextArea.<clinit>(JSyntaxTextArea.java:54)
```

Reply:
No the static theme isn't necessary. I just tried to be as close to the previous implementation as possible.